### PR TITLE
Add a milestone for stars

### PR DESCRIPTION
**Pull request by tianxiemaochiyu**
Add a milestone for stars by dynamically updating Emoji in the Description of repository.
#2

### CODE REVIEW

**Comment by PiyushSuthar** [comment]
You have misspelled `star` as `start`.

File: index.js [hunk 1]
```diff
@@ -27,7 +43,10 @@ const checkAndRenameRepo = async () => {
         const stars = await getRepoStars()
 
         if (lastStars !== stars) {
-            await renameRepo(`This-Repo-Has-${stars}-Stars`)
+            await renameRepo({
+                name: `This-Repo-Has-${stars}-Stars`,
+                description: `Yes, it's true ${getMilestone(starts)}`
```
Comment (PiyushSuthar) [inline index.js:48]: You have misspelled `star` as `start`.

Comment (RealPeha) [reply index.js:48]: @tianxiemaochiyu, don't worry about it, I'll fix it

**Comment by RealPeha** [comment]
@tianxiemaochiyu very cool, I will make minor edits and merge

**Pull request merged**